### PR TITLE
Add rate limit in HDK

### PR DESCRIPTION
**Pull request by muyuncaigeotab**

### CODE REVIEW

File: src/hardware/addon-protocols/rs232-usb.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ Issued by the external device whenever it requires Third-Party Data to be saved
 
 ### Msg Type 0x82: Free Format Third-Party Data
 
-Issued by the external device whenever it wants Third-Party Data to be saved on the GO device in a free format (1 to 27 bytes) that will be saved into MyGeotab as Custom Data.
+Issued by the external device whenever it wants Third-Party Data to be saved on the GO device in a free format (1 to 27 bytes) that will be saved into MyGeotab as Custom Data. Rate limit is 500 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX
```
Comment (Stephen-F) [inline src/hardware/addon-protocols/rs232-usb.md:259]: If you exceed the rate limit, the GO device will stop accepting Third-Party data from the IOX.

File: src/hardware/developing-an-iox/protocol.md [hunk 2]
```diff
@@ -229,9 +229,9 @@ Sent from the IOX to the GO device when the IOX wants create a log that cannot f
 
 | Parameter Type | Description |
 | --- | --- |
-| 0 | Third Party Free Format Data |
+| 0 | Third Party Free Format Data. Rate limit is 500 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX |
 | 1 | Reserved |
-| 2 | Bluetooth Record |
+| 2 | Bluetooth Record. Rate limit is 1200 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX |
```
Comment (Stephen-F) [inline src/hardware/developing-an-iox/protocol.md:234]: If you exceed the rate limit, the GO device will stop accepting Third-Party data from the IOX.

File: src/hardware/developing-an-iox/protocol.md [hunk 3]
```diff
@@ -229,9 +229,9 @@ Sent from the IOX to the GO device when the IOX wants create a log that cannot f
 
 | Parameter Type | Description |
 | --- | --- |
-| 0 | Third Party Free Format Data |
+| 0 | Third Party Free Format Data. Rate limit is 500 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX |
```
Comment (Stephen-F) [inline src/hardware/developing-an-iox/protocol.md:232]: If you exceed the rate limit, the GO device will stop accepting Bluetooth data from the IOX.

File: src/hardware/addon-protocols/ble.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ title: Add-On Protocol - BLE
 
 External devices can communicate with the Geotab GO device through the Third-Party Bluetooth Low Energy (BLE) protocol below. The hardware interface will be the [IOX-BT](https://www.geotab.com/documentation/iox-bt/).
 
-The IOX-BT is a read-only BLE sensor hub that supports up to 200 in-range beacons and will detect in/out of range for any Bluetooth beacon with a public MAC Address. However, sending any other data points requires the beacon to conform to the below specified Geotab BLE protocol.
+The IOX-BT is a read-only BLE sensor hub that supports up to 200 in-range beacons and will detect in/out of range for any Bluetooth beacon with a public MAC Address. However, sending any other data points requires the beacon to conform to the below specified Geotab BLE protocol. Rate limit is 1200 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX
```
Comment (Stephen-F) [inline src/hardware/addon-protocols/ble.md:9]: Missing period.

File: src/hardware/addon-protocols/rs232-usb.md (hunk 1 above)
Comment (Stephen-F) [inline src/hardware/addon-protocols/rs232-usb.md:259]: Missing period.

File: src/hardware/introduction.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ permalink: /hardware/introduction/
...
+When using Geotab's Hardware Add-On protocols, it is important not to send excessive amounts of data to the GO device.  This can be assured by only sending data when it is completely necessary and avoiding the sending of repetitive values. The GO device will stop recording data from the Hardware Add-On device if excessive Add-On data is detected. When this condition occurs, "Status data no longer being saved due to excessive data" or "Third-party data no longer being saved due to excessive data" will appear in the Fault Data for the GO device within MyGeotab. In order to re-enable data logging for the Hardware Add-On, the GO device will need to be completely disconnected from power and then powered back up again.
```
Comment (Stephen-F) [inline src/hardware/introduction.md:11]: There is also a FAULT_BLUETOOTH_DATA_LOGGING_TURNED_OFF.

Comment (muyuncaigeotab) [reply src/hardware/introduction.md:11]: Hi @Stephen-F , no entry in emGoPid matches FAULT_BLUETOOTH_DATA_LOGGING_TURNED_OFF, code 376. That is a mistake, correct? I can help to correct it. Would you like customers to see "Bluetooth data no longer being saved due to excessive data" on the UI?

**Pull request merged**